kreuzberg/_api/main.py

```diff
@@ -223,6 +223,13 @@ async def handle_files_upload(  # noqa: PLR0913
     ocr_backend: Literal["tesseract", "easyocr", "paddleocr"] | None = None,
     auto_detect_language: str | bool | None = None,
     pdf_password: str | None = None,
+    extract_images: str | bool | None = None,
+    ocr_extracted_images: str | bool | None = None,
+    image_ocr_backend: Literal["tesseract", "easyocr", "paddleocr"] | None = None,
```

Comment:
Let's create an ImageOCRConfig dataclass instead of. Passing 5 or 6 new parameters to the config.

Reply:
We also have to support proper OCR configs for these, so this dataclass should extend or share a base class with the other OCR configs. This requires further planning